src/components/ActionBar/style.css

```diff
@@ -0,0 +1,15 @@
+.base {
```

Comment:
This makes sense.

As we mentioned offline, it might make more sense to call 'themes' instead 'common'.

And then with that, the ordering is a little off, yeah. It would make more sense for any style to override the common style, by default. I guess just ordering 'common' first is fine for now, though